code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -157,16 +157,20 @@ namespace WijmoProvider.Feature {
             if (action.dataItem === undefined) return;
 
             const binding = this._grid.provider.getColumn(action.col).binding;
-            const oldValue = this._grid.features.dirtyMark.getOldValue(
-                action.row,
-                binding
-            );
-            this._triggerEventsFromColumn(
-                action.row,
-                binding,
-                oldValue,
-                action._newState
-            );
+
+            // we don't want to redo on GridRemoveRowAction
+            if (typeof action._oldState !== 'object') {
```

Comment:
Following @rugoncalves suggestion, this one is mine :p

```
private _redoingActionHandler(
    s: wijmo.undo.UndoStack,
    e: wijmo.undo.UndoActionEventArgs
): void {
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    const action: any = e.action;
    
    // we only want to redo on GridEditAction
    // we don't want to redo on GridRemoveRowAction
    if (action.dataItem !== undefined && typeof action._oldState !== 'object') {
        const binding = this._grid.provider.getColumn(action.col).binding;
        const oldValue = this._grid.features.dirtyMark.getOldValue(
            action.row,
            binding
        );
        this._triggerEventsFromColumn(
            action.row,
            binding,
            oldValue,
            action._newState
        );
    }
}
```

Reply:
fixed